core.multiview/src/org/netbeans/core/multiview/actions/SplitDocumentHorizontallyAction.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.core.multiview.actions;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.Action;
+import javax.swing.JSplitPane;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionRegistration;
+import org.openide.util.NbBundle;
+import org.openide.windows.TopComponent;
+import org.openide.windows.WindowManager;
+
+/**
+ *
+ * @author Christian Lenz (Chrizzly)
+ */
+@ActionID(
+    category = "Tools",
+    id = "org.netbeans.core.multiview.SplitDocumentHorizontallyAction"
+)
+@ActionRegistration(
+    displayName = "#LBL_ValueSplitHorizontal"
+)
+@ActionReference(path = "Shortcuts", name = "DOS-H")
```

Comment:
Also consider creating `@ActionReference` for placement in `Menu/Windows`. Note that the registered singletons (unlike instance created in window popup menus) do not update their enable flag according to the current TC.